### PR TITLE
Update default scopes and add promptForLogin

### PR DESCRIPTION
**Pull request by tcobbs-bentley**

### CODE REVIEW

File: Sources/ITwinMobile/ITMOIDCAuthorizationClient.swift [hunk 1]
```diff
@@ -116,6 +120,8 @@ open class ITMOIDCAuthorizationClient: NSObject, ITMAuthorizationClient, OIDAuth
     }
     internal static var currentAuthorizationFlow: OIDExternalUserAgentSession?
     private var loadStateActive = false
+    private let defaultScopes = "projects:read imodelaccess:read itwinjs organization profile email imodels:read realitydata:read savedviews:read savedviews:modify itwins:read openid offline_access"
+    private let defaultRedirectUri = "imodeljs://app/signin-callback"
```
Comment (toddsouthenbentley) [inline Sources/ITwinMobile/ITMOIDCAuthorizationClient.swift:124]: Why was this constant introduced? It seems to only be used once.

Comment (tcobbs-bentley) [reply Sources/ITwinMobile/ITMOIDCAuthorizationClient.swift:124]: With the much longer new scope, I felt that making defaultScopes a constant made it easier to read. To be consistent, I also made defaultRedirectUri a constant.

Comment (toddsouthenbentley) [reply Sources/ITwinMobile/ITMOIDCAuthorizationClient.swift:124]: Maybe `defaultIssuerUrl` should be added too then?

Comment (tcobbs-bentley) [reply Sources/ITwinMobile/ITMOIDCAuthorizationClient.swift:124]: That one relies on a variable, so isn't a constant value.

**Review by toddsouthenbentley — APPROVED** [review]

**Pull request merged**